.github/dependabot.yml

```
@@ -9,3 +9,7 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    labels:
+      - "dependencies"
+      - "javascript"
+      - "team-security"
```

or perhaps

```suggestion
      - "team-mobile-client"
```

is more appropriate here